tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml

```diff
@@ -8,14 +8,14 @@ jobs:
   pool: 'AMD-GPU'
   timeoutInMinutes: 180
 
-  # gid of video and render group on gcr-openpai-35 and -36
+  # gid of video and render group on gcramdrr1-mi100-085 and -86
```

Comment:
>gid [](http://example.com/codeflow?start=4&length=3)

nit - get

Reply:
The gid means group id, This comment means that the vaule of the variable video is it's group id.